code/game/machinery/vending.dm

```diff
@@ -1723,6 +1860,15 @@
 	product_ads = "Put your foot down!;One size fits all!;IM WALKING ON SUNSHINE!;No hobbits allowed.;NO PLEASE WILLY, DONT HURT ME- *BZZT*"
 	products = list(/obj/item/clothing/shoes/black = 10,/obj/item/clothing/shoes/brown = 10,/obj/item/clothing/shoes/blue = 10,/obj/item/clothing/shoes/green = 10,/obj/item/clothing/shoes/yellow = 10,/obj/item/clothing/shoes/purple = 10,/obj/item/clothing/shoes/red = 10,/obj/item/clothing/shoes/white = 10,/obj/item/clothing/shoes/sandal=10)
 	contraband = list(/obj/item/clothing/shoes/orange = 5)
```

Comment:
Tambien podrias añadir aqui mas contrabando.
Y tambien me pareceria buena idea quitar el wand of nothing del autrodrobe que no pega mucho en un dispensador de ropas.

Reply:
Añadere unas jackboots y borrare la wand

Reply:
Los jackboots ya salen sin ser contrabando en el autodrobe, si los pones de contrabando quita los del autodrobe.